### PR TITLE
Lvjav442 20

### PR DESCRIPTION
**Pull request by VOleh**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,6 +55,22 @@
 			<artifactId>mysql-connector-java</artifactId>
 			<version>8.0.15</version>
 		</dependency>
```
Comment (NazarRyvak) [inline pom.xml]: remove dependency

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Document.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.softserve.ita.java442.cityDonut.model;
+
+import lombok.Builder;
+import lombok.Data;
+import javax.persistence.*;
+
+@Entity
+@Builder
+@Data
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Document.java:12]: add @AllArgsConstructor and @NoArgsConstructor

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Donate.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "date",nullable = false)
+    private LocalDateTime date;
+
+    @Column(name = "sum",nullable = false)
+    private double sum;
+
```
Comment (martavoloshyn) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Donate.java]: change CascadeType.ALL

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Document.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+@Table(name="document")
+public class Document {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "name")
+    private double sum;
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Document.java]: change field name

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Document.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+    @ManyToOne(cascade = {CascadeType.ALL}, fetch = FetchType.LAZY)
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Document.java:26]: change cascade

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Donate.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+import lombok.Builder;
+import lombok.Data;
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Builder
+@Data
+@Table(name="donate")
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Donate.java]: add @AllArgsConstructor and @NoArgsConstructor

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Document.java [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+import javax.persistence.*;
+
+@Entity
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "document")
+public class Document {
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Document.java:17]: remove empty row

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Donate.java [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+import java.time.LocalDateTime;
+
+@Entity
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "donate")
+public class Donate {
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Donate.java:18]: remove empty row

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Donate.java [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    @Column(name = "sum", nullable = false)
+    private double sum;
+
+    @ManyToOne(cascade = {
+            CascadeType.DETACH, CascadeType.MERGE,
+            CascadeType.PERSIST, CascadeType.REFRESH}, fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+}
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Donate.java:35]: where is project field?

**Pull request closed without merging**